src/components/sidebar-toggle-button/SidebarToggleButton.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../styles/variables';
+
+.bdl-SidebarToggleButton,
+.bdl-SidebarToggleButton:hover,
```

Comment:
Do all of the rules need to be re-applied for each pseudo-class? We can also avoid repeating the class name:

```
.bdl-SidebarToggleButton {
    &,
    &:hover,
    &:focus {
        // style rules
    }
}